yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/aml/RemoteAmlService.java

```diff
@@ -1,54 +1,57 @@
 package com.yoti.api.client.spi.remote.call.aml;
 
+import static java.net.HttpURLConnection.HTTP_BAD_REQUEST;
+import static java.net.HttpURLConnection.HTTP_INTERNAL_ERROR;
+import static java.net.HttpURLConnection.HTTP_UNAUTHORIZED;
```

Comment:
\- trivial - I've to admit avoiding static imports improves, IMO, the readability. When u look at someone else's code reading 

CostantClass.CONSTANT instead of CONSTANT makes u realise immediately if CostantClass comes from a 3rd party lib or an internal. 

Free to ignore this comment though